clang/test/lab1/borovkov_sergey/tests.cpp

```diff
@@ -0,0 +1,212 @@
+// RUN: split-file %s %t
+
+// RUN: %clang_cc1 -load %llvmshlibdir/renamedfilePlugin%pluginext\
+// RUN: -add-plugin rename\
+// RUN: -plugin-arg-rename type=var\
+// RUN: -plugin-arg-rename cur-name=a\
+// RUN: -plugin-arg-rename new-name=new_var %t/rename_var.cpp
+// RUN: FileCheck %s < %t/rename_var.cpp --check-prefix=VAR
+
+// VAR: int func() {
```

Comment:
Please move checks closer to input code samples.

Reply:
Done.

Reply:
Sorry if I wasn't clear. I meant smth like:

```
//--- file1.cpp

// RUN: cmd ... %t/file1.cpp
// FileCheck %t/file1.cpp

// CHECK: ..

int func1...
```